Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/TypeScriptExpressApp.njsproj

```diff
@@ -31,9 +28,9 @@
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
   <ItemGroup>
-    <TypeScriptCompile Include="app.ts" />
-    <TypeScriptCompile Include="routes\index.ts" />
-    <TypeScriptCompile Include="routes\user.ts" />
+    <Compile Include="app.ts" />
+    <Compile Include="routes\index.ts" />
+    <Compile Include="routes\user.ts" />
     <Content Include="tsconfig.json" />
```

Comment:
If `<Compile>` marks the file that trigger a rebuild, it seems like tsconfig.json should be `<Compile>`.

Reply:
Good question, I tried it and it works. It also have the side effect of always building the project regardless if any file has been modified or not. I didn't detect any other side effect on the couple of tests I executed. I will change the templates.

Reply:
Change of plans, Compile no longer decides if the file is "dirty". Instead, we removed that logic and make the project "dirty" all the time, to allow either the SDK or tsc to build it.